src/lib/units/year.js

```diff
@@ -9,6 +9,18 @@ import toInt from '../utils/to-int';
 
 // FORMATTING
 
+addFormatToken('Y', 0, 0, function () {
+    var y = this.year();
+    if (y < 0) {
+        return y;
+    } else if (y <= 9999) {
+        return y;
+    } else {
+        // force plus for longer years.
+        return '+' + y;
+    }
```

Comment:
This could just be `return y < 1e4 ? y : '+' + y;`

Also, do we need to *always* return a string, and do years between 0 and 999 need to be zero-padded to 4-digits?  (not sure)